rules/xcodeproj.bzl

```diff
@@ -652,11 +652,19 @@ def _xcodeproj_impl(ctx):
     proj_settings_debug = {
         "GCC_PREPROCESSOR_DEFINITIONS": "DEBUG",
         "SWIFT_ACTIVE_COMPILATION_CONDITIONS": "DEBUG",
+        "BAZEL_CUSTOM_BUILD_OPTIONS": ctx.attr.custom_build_options_by_config.get("Debug", []),
     }
+
+    # For release config only:
```

Comment:
are the paths and compiler options we're passing for debug config also valid for release?

Reply:
I think so. I was able to build ver complicate app for release in xcode.